### ISSUE TITLE
Add extension methods to `VoiceResponse` and `MessagingResponse` to convert to `TwiMLResult`

### ISSUE THREAD

**Issue by Swimburger** [twilio-labs/twilio-aspnet#61]
We could add extension methods to make the TwiML generation code more eloquent.
Consider this code:
```csharp
app.MapPost("/submit", () =>
{
    var voiceResponse = new VoiceResponse()
        .Say($"You pressed 1. Good choice. Goodbye.");
    return Results.Extensions.TwiML(voiceResponse);
});
```
Which could become:
```csharp
app.MapPost("/submit",  () => 
    new VoiceResponse()
        .Say($"You pressed 1. Good choice. Goodbye.")
        .ToTwiMLResult()
);
```

with a convenient extension method.

**Comment by dprothero** [comment]
@Swimburger Seems cool. It'd be amazing if you could just return a `VoiceResponse` object and it automatically converts it to a `TwiMLResult` for you without needing any function, however. Would that just involve overriding the `.ToString()` method of `VoiceResponse`, or something else perhaps?

**Comment by Swimburger** [comment]
@dprothero that would get the XML into the body, but wouldn't set the content-type header.

**Comment by Swimburger** [comment]
Implemented by #101.
